### PR TITLE
feat: add hero component to home page

### PR DESCRIPTION
**Pull request by braindrain9**
Screenshot 2020-04-27 at 12 30 56
Screenshot 2020-04-27 at 12 31 06
Screenshot 2020-04-26 at 20 38 49

### CODE REVIEW

**Comment by olegdolenchuk** [comment]
Screenshot 2020-04-26 at 22 15 11
I think the image should be located to the right

**Comment by braindrain9** [comment]
thanks @olegdolenchuk 👍 missed that. 

updated 👍

File: src/shared/media-types.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 enum MediaTypes {
   MOBILE = '576',
-  TABLET_M = '768',
-  TABLET_L = '1024',
+  TABLET = '768',
```
Comment (braindrain9) [inline src/shared/media-types.ts:3]: updated TABLET media types naming for 2 reasons:
1) `992px` was missing
2) tabletMMax is not very comfortable to use

**Comment by olegdolenchuk** [comment]
Screenshot 2020-04-27 at 10 23 37
also noticed that there is no button

**Comment by braindrain9** [comment]
good catch @olegdolenchuk  👍 updated 👍

**Pull request merged**